tensorflow/contrib/rnn/python/ops/gru_ops.py

```diff
@@ -129,13 +130,21 @@ class GRUBlockCell(rnn_cell_impl.RNNCell):
 
   """
 
-  def __init__(self, cell_size):
+  @deprecated_args(None, "cell_size is deprecated, use num_units instead",
+                   "cell_size")
+  def __init__(self, num_units=None, cell_size=None):
     """Initialize the Block GRU cell.
 
     Args:
-      cell_size: int, GRU cell size.
+      num_units: int, The number of units in the GRU cell
     """
-    self._cell_size = cell_size
+    if cell_size is not None:
```

Comment:
if (cell_size is None) == (num_units is None):
  raise ValueError("Exactly one of num_units or cell_size must be provided.")

Reply:
@ebrevdo Thanks. Indeed that is much cleaner. Updated.